data-prepper-plugins/csv-processor/src/main/java/org/opensearch/dataprepper/plugins/csvinputcodec/CsvCodecConfig.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.dataprepper.plugins.source.codec;
+package org.opensearch.dataprepper.plugins.csvinputcodec;
```

Comment:
I think a better package name would be either of the following:

```
org.opensearch.dataprepper.plugins.csv.codec
```

or

```
org.opensearch.dataprepper.plugins.codec.csv
```

The latter fits the current package name for the `csv-processor` project.